Tests/run_tests.py

```diff
@@ -7,9 +7,7 @@
 This will find all modules whose name is "test_*.py" in the test
 directory, and run them.  Various command line options provide
 additional facilities.
-
```

Comment:
You have accidentally remove this (and two more) blank line from the module's docstring.